src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ActiveRequestsPage/index.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import * as styles from './styles.less';
+import {
+    Button,
+    IconButton,
+    DeleteIcon,
+    EditIcon,
+    Spinner,
+    ErrorMessage,
+} from '@equinor/fusion-components';
+import PersonnelRequest from '../../../../../../models/PersonnelRequest';
+import { useAppContext } from '../../../../../../appContext';
+import SortableTable from '../components/SortableTable';
+import columns from './columns';
+
+const ActiveRequestsPage: React.FC = () => {
+    const [activeRequests, setActiveRequests] = React.useState<PersonnelRequest[]>([]);
+    const [isFetching, setIsFetching] = React.useState<boolean>(false);
+    const [error, setError] = React.useState(null);
+    const [selectedRequests, setSelectedRequests] = React.useState<PersonnelRequest[]>([]);
+    const { apiClient } = useAppContext();
+    const getRequestsAsync = async () => {
+        setIsFetching(true);
+        setError(null);
+        try {
+            const response = await apiClient.getPersonnelRequestsAsync('123', '123'); //TESTING VALUES
+            const activeRequests = response.data.value.filter(
```

Comment:
This should probably be handled by the backend with an OData filter

Reply:
It should, but as of right now it's not implemented. Should I just remove the frontend filter?

Reply:
The backend just returns bogus data. We should set up the frondend as if the backend was implemented correctly

Reply:
Changed to OData filter in url